axelrod/tests/strategies/test_zero_determinant.py

```diff
@@ -317,3 +317,46 @@ def test_strategy(self):
 
         actions = [(C, D), (D, C), (D, D), (D, C), (D, D), (D, C)]
         self.versus_test(opponent=axelrod.CyclerDC(), expected_actions=actions, seed=5)
+
+
+class TestAdaptiveZeroDet(TestPlayer):
+    """
+    Test the AdaptiveZeroDet strategy.
+    """
+
+    name = "AdaptiveZeroDet: 0.125, 0.5, 3, C"
+    player = axelrod.AdaptiveZeroDet
+    expected_classifier = {
+        "memory_depth": float('inf'),
+        "stochastic": True,
+        "makes_use_of": set(['game']),
+        "long_run_time": False,
+        "inspects_source": False,
+        "manipulates_source": False,
+        "manipulates_state": False,
+    }
+
```

Comment:
Could we add few tests for the two specific methods: `_adjust_parameters` and `score_last_round` (just running the methods and checking that the attributes have changed accordingly?